roles/ideascube/files/ideascube_leftover_files.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Run ideascube clean leftover-files
+RefuseManualStart=no
+RefuseManualStop=yes
+ 
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/ideascube clean leftover-files
```

Comment:
Thanks for the hint ! So in that case we could say that systemd has to wait, let say 8min before checking if `ExecStart=/usr/bin/ideascube clean leftover-files` did succeed correctly. Meaning the files were cleaned up. 

Correct ?